src/editor/bin/Input.re

```diff
@@ -32,58 +37,68 @@ let charToCommand = (codepoint: int, mods: Modifier.t) => {
 
   let key =
     keyPressToString(~shiftKey=false, ~altKey, ~ctrlKey, ~superKey, char);
-  Some(key);
+  let shouldOniListen = isOniModifier(~altKey, ~ctrlKey, ~superKey, ~key);
+  Some((key, shouldOniListen));
 };
 
 let keyPressToCommand =
-    (key: Key.t, buttonState: ButtonState.t, mods: Modifier.t) =>
-  if (buttonState == GLFW_PRESS || buttonState == GLFW_REPEAT) {
-    /* If ctrl is pressed, it's a non printable character, not handled by charMods - so we handle any character */
-    let key =
-      if (Modifier.isControlPressed(mods)) {
+    ({shiftKey, altKey, ctrlKey, superKey, key, _}: Events.keyEvent) => {
+  let keyString =
+    ctrlKey
+      ? /**
+        TODO:
+        Revery's toString method returns lower case
```

Comment:
Is this comment still applicable, now that we are capitalizing the return value?